target/linux/ath79/dts/qca9557_kt_dw02-412h.dtsi

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "KT DW02-412H (64M)";
+	compatible = "kt,dw02-412h-64m", "qca,qca9557";
```

Comment:
Drop these two lines for obvious reasons.

Reply:
I basically edited this file from .dts file, and I made a mistake with saving the modifications on this file before pushing it.
Sorry for making you check these again...